Wikipedia/Code/WMFExploreCollectionViewController.m

```diff
@@ -569,6 +568,7 @@ - (void)traitCollectionDidChange:(nullable UITraitCollection *)previousTraitColl
     [self resetLayoutCache];
     [super traitCollectionDidChange:previousTraitCollection];
     [self registerForPreviewingIfAvailable];
+    [self sizeNotificationHeader];
```

Comment:
Good catch - the resize on orientation change should be fixable with a `[self sizeNotificationHeader];` call within `- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator` underneath `[self showNotificationHeader];`